### DISCUSSION TITLE
Does Milvus support nvidia jetson xavier (arm architecture)?

### DISCUSSION THREAD

**Discussion by 17120033** [milvus-io/milvus#4160] [Q&A and General discussion] [answered]
i need to do embeddings similarity search on nvidia jetson xavier,  Does Milvus support nvidia jetson xavier (arm architecture)?

**Comment by shiyu22** [comment, marked as answer]
About GPU support, you can follow [this doc](https://milvus.io/docs/v0.11.0/operational_faq.md).

How can I know whether my GPU is supported by Milvus?
Milvus supports CUDA architecture 6.0 or later.
